wme-sitebuilder/Plugins/PaymentGateways/PayPal.php

```diff
@@ -134,7 +134,9 @@ public function action__woocommerce_paypal_payments_built_container( $container
 			return;
 		}
 
+		/** @var \WooCommerce\PayPalCommerce\Onboarding\Render\OnboardingOptionsRenderer $onboarding */
 		$onboarding = $container->get( 'onboarding.render' );
+		/** @var \WooCommerce\PayPalCommerce\WcGateway\Settings\Settings $settings */
```

Comment:
There are more advanced solutions than adding individual `@var` tags.

Although it takes only a few minutes to do it!